containers/src/Kokkos_DualView.hpp

```diff
@@ -441,8 +441,12 @@ class DualView : public ViewTraits<DataType, Properties...> {
         }
       }
     }
-#ifdef KOKKOS_COMPILER_INTEL
-    __builtin_unreachable();
+    // Got:
+    //  error #940-D: missing return statement at end of non-void function
+    // so I added '__builtin_unreachable' and now I get the calling host
+    // function from host device function.
+#if defined(KOKKOS_COMPILER_NVCC) && KOKKOS_COMPILER_NVCC == 1100
+    Kokkos::abort("This should never happen.");
```

Comment:
Decide what to do here.

Reply:
I decided to use `diag_suppress=940` at CLI level to avoid this (see comment below).